nix/modules/cockroachdb.nix

```diff
@@ -287,6 +287,18 @@ in
     users.groups = lib.optionalAttrs (cfg.group == "cockroachdb") {
       cockroachdb.gid = config.ids.gids.cockroachdb;
     };
+    security.sudo.extraRules = [
+      {
+        runAs = "root";
+        groups = [ "cockroachdb" ];
+        commands = [
+          {
+            command = "/run/current-system/sw/bin/systemctl restart cockroachdb";
```

Comment:
I've not had to restart a cockroach node without making a config change.

Reply:
It would help if you use `sudo systemctl restart cockroachdb` not `systemctl restart cockroachdb`.  Also, the `kld-mgr` will help you do that, the a normal user will be restricted from using `systmectl`, so we can prevent unintentional stop service.